[2024-06-13 08:02:44,510] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.02, 127.0.0.1:60708]
[2024-06-13 08:02:45,509] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:60699]
[2024-06-13 08:02:45,630] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:60699]
[2024-06-13 08:02:45,672] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:60699]
[2024-06-13 08:02:45,690] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:60711]
[2024-06-13 08:02:45,760] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:60699]
[2024-06-13 08:02:45,766] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:60711]
[2024-06-13 08:02:45,767] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:60709]
[2024-06-13 08:02:45,914] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:02:45 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 08:02:45,925] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:60709]
[2024-06-13 08:02:45,967] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:60699]
[2024-06-13 08:12:37,740] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.15, 127.0.0.1:61596]
[2024-06-13 08:12:37,855] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:61599]
[2024-06-13 08:12:37,932] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:61599]
[2024-06-13 08:12:38,023] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:12:38 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 08:12:38,034] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:61596]
[2024-06-13 08:12:38,132] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:61596]
[2024-06-13 08:12:42,326] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /admin/ 302 [0.26, 127.0.0.1:61596]
[2024-06-13 08:12:42,403] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/login/?next=/admin/ 200 [0.07, 127.0.0.1:61596]
[2024-06-13 08:12:42,452] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:61599]
[2024-06-13 08:12:42,573] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:61596]
[2024-06-13 08:12:44,936] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:12:44 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /admin/login/ POST
[2024-06-13 08:12:44,947] [runserver.py:147] [runserver:log_action] [INFO]- HTTP POST /admin/login/?next=/admin/ 302 [0.35, 127.0.0.1:61596]
[2024-06-13 08:12:45,864] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.91, 127.0.0.1:61596]
[2024-06-13 08:12:46,022] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:61596]
[2024-06-13 08:12:53,167] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.13, 127.0.0.1:61596]
[2024-06-13 08:12:53,198] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.15, 127.0.0.1:61597]
[2024-06-13 08:12:53,204] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/css/base.css?_=2.7 200 [0.03, 127.0.0.1:61596]
[2024-06-13 08:12:53,209] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/css/base.css?_=2.7 200 [0.03, 127.0.0.1:61638]
[2024-06-13 08:12:53,214] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/theme/simpleui.css 200 [0.03, 127.0.0.1:61639]
[2024-06-13 08:12:53,218] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/css/changelists.css?_=20180905 200 [0.04, 127.0.0.1:61640]
[2024-06-13 08:12:53,223] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/js/language.js?_=2.1.2 200 [0.04, 127.0.0.1:61641]
[2024-06-13 08:12:53,228] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/css/responsive.css?_=20180905 200 [0.03, 127.0.0.1:61597]
[2024-06-13 08:12:53,233] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/locale/en-us.js?_=2.1.2 200 [0.03, 127.0.0.1:61596]
[2024-06-13 08:12:53,237] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/css/fonts.css 200 [0.03, 127.0.0.1:61638]
[2024-06-13 08:12:53,241] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/locale/zh-hans.js 200 [0.03, 127.0.0.1:61639]
[2024-06-13 08:12:53,245] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/js/cookie.js 200 [0.03, 127.0.0.1:61640]
[2024-06-13 08:12:53,269] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:61641]
[2024-06-13 08:12:53,275] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/vendor/jquery/jquery.js 200 [0.05, 127.0.0.1:61597]
[2024-06-13 08:12:53,281] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/jquery.init.js 200 [0.05, 127.0.0.1:61596]
[2024-06-13 08:12:53,287] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/core.js 200 [0.05, 127.0.0.1:61638]
[2024-06-13 08:12:53,291] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/admin/RelatedObjectLookups.js 200 [0.05, 127.0.0.1:61639]
[2024-06-13 08:12:53,298] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/actions.js 200 [0.05, 127.0.0.1:61640]
[2024-06-13 08:12:53,304] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/urlify.js 200 [0.03, 127.0.0.1:61641]
[2024-06-13 08:12:53,310] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/prepopulate.js 200 [0.03, 127.0.0.1:61597]
[2024-06-13 08:12:53,315] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/vendor/xregexp/xregexp.js 200 [0.03, 127.0.0.1:61596]
[2024-06-13 08:12:53,318] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/vendor/jquery/jquery.min.js 200 [0.03, 127.0.0.1:61638]
[2024-06-13 08:12:53,323] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/automatic/dicts.js 200 [0.03, 127.0.0.1:61639]
[2024-06-13 08:12:53,328] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/automatic/segment.js 200 [0.03, 127.0.0.1:61640]
[2024-06-13 08:12:53,333] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/simpleui-x/js/axios.min.js 200 [0.03, 127.0.0.1:61641]
[2024-06-13 08:12:53,353] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/fonts/Roboto-Regular-webfont.woff 200 [0.00, 127.0.0.1:61641]
[2024-06-13 08:12:53,364] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/img/sorting-icons.svg 200 [0.01, 127.0.0.1:61641]
[2024-06-13 08:12:53,375] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/fonts/Roboto-Light-webfont.woff 200 [0.00, 127.0.0.1:61641]
[2024-06-13 08:12:53,381] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/fonts/Roboto-Bold-webfont.woff 200 [0.01, 127.0.0.1:61640]
[2024-06-13 08:13:11,299] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:61640]
[2024-06-13 08:13:11,398] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:61640]
[2024-06-13 08:13:11,412] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:61640]
[2024-06-13 08:13:11,461] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:61680]
[2024-06-13 08:13:11,465] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:61680]
[2024-06-13 08:13:11,568] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:61640]
[2024-06-13 08:13:11,575] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:61641]
[2024-06-13 08:13:11,722] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:13:11 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 08:13:11,732] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:61641]
[2024-06-13 08:13:11,767] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:61640]
[2024-06-13 08:13:30,369] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:61640]
[2024-06-13 08:13:30,377] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:61680]
[2024-06-13 08:13:30,456] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:61640]
[2024-06-13 08:13:30,494] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:61640]
[2024-06-13 08:13:30,539] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:61716]
[2024-06-13 08:13:30,542] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:61716]
[2024-06-13 08:13:30,643] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:61640]
[2024-06-13 08:13:30,651] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:61641]
[2024-06-13 08:13:30,767] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:13:30 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 08:13:30,778] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:61641]
[2024-06-13 08:13:30,822] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:61640]
[2024-06-13 08:42:48,720] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:50384]
[2024-06-13 08:42:48,728] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:61716]
[2024-06-13 08:42:48,819] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:50384]
[2024-06-13 08:42:48,834] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:50384]
[2024-06-13 08:42:48,880] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:50387]
[2024-06-13 08:42:48,884] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:50387]
[2024-06-13 08:42:48,971] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:50384]
[2024-06-13 08:42:49,009] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:50384]
[2024-06-13 08:42:49,147] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:42:49 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 08:42:49,177] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:50384]
[2024-06-13 08:42:49,208] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:50385]
[2024-06-13 08:42:54,396] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 268, in CapitalExpenditure_Summary
    if key in headermodel_CapitalExpenditure.keys():
AttributeError: 'tuple' object has no attribute 'keys'
[2024-06-13 08:42:54,403] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 08:42:54 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 08:42:54,406] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.09, 127.0.0.1:50385]
[2024-06-13 09:48:00,608] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:64723]
[2024-06-13 09:48:00,845] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:64723]
[2024-06-13 09:48:00,906] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:64723]
[2024-06-13 09:48:00,967] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64727]
[2024-06-13 09:48:01,005] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64727]
[2024-06-13 09:48:01,091] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:64723]
[2024-06-13 09:48:01,104] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:64723]
[2024-06-13 09:48:01,309] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:48:01 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:48:01,320] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:64723]
[2024-06-13 09:48:01,348] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:64724]
[2024-06-13 09:48:06,504] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 268, in CapitalExpenditure_Summary
    print(headermodel_CapitalExpenditure.keys())
AttributeError: 'tuple' object has no attribute 'keys'
[2024-06-13 09:48:06,509] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:48:06 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:48:06,513] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.38, 127.0.0.1:64724]
[2024-06-13 09:53:12,856] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:58500]
[2024-06-13 09:53:12,970] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:58500]
[2024-06-13 09:53:12,982] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:58500]
[2024-06-13 09:53:12,991] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:58503]
[2024-06-13 09:53:13,024] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:58503]
[2024-06-13 09:53:13,042] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:58500]
[2024-06-13 09:53:13,047] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:58501]
[2024-06-13 09:53:13,182] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:53:13 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:53:13,193] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:58501]
[2024-06-13 09:53:13,227] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:58500]
[2024-06-13 09:53:17,183] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 245, in CapitalExpenditure_Summary
    print(headermodel_CapitalExpenditure.keys())
AttributeError: 'tuple' object has no attribute 'keys'
[2024-06-13 09:53:17,189] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:53:17 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:53:17,191] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.46, 127.0.0.1:58500]
[2024-06-13 09:54:41,203] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:58641]
[2024-06-13 09:54:41,296] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:58641]
[2024-06-13 09:54:41,314] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:58641]
[2024-06-13 09:54:41,329] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:58646]
[2024-06-13 09:54:41,358] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:58646]
[2024-06-13 09:54:41,378] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:58641]
[2024-06-13 09:54:41,382] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:58640]
[2024-06-13 09:54:41,508] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:54:41 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:54:41,519] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:58640]
[2024-06-13 09:54:41,563] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:58641]
[2024-06-13 09:54:45,014] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:54:45 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:54:45,042] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:58641]
[2024-06-13 09:55:02,699] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.66, 127.0.0.1:58691]
[2024-06-13 09:55:02,864] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.06, 127.0.0.1:58691]
[2024-06-13 09:55:02,917] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:58691]
[2024-06-13 09:55:03,047] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:58691]
[2024-06-13 09:55:17,869] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:55:17 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:55:17,879] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:58691]
[2024-06-13 09:56:34,113] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:56:34 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:56:34,130] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:58846]
[2024-06-13 09:57:39,085] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:57:39 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:57:39,104] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:58961]
[2024-06-13 09:59:03,879] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 09:59:03 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 09:59:03,895] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:57391]
[2024-06-13 10:01:01,201] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:57614]
[2024-06-13 10:01:01,278] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:57614]
[2024-06-13 10:01:01,312] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:57614]
[2024-06-13 10:01:01,320] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:57617]
[2024-06-13 10:01:01,352] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:57617]
[2024-06-13 10:01:01,371] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:57614]
[2024-06-13 10:01:01,377] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:57615]
[2024-06-13 10:01:01,508] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:01:01 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:01:01,519] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:57615]
[2024-06-13 10:01:01,564] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:57614]
[2024-06-13 10:01:04,905] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 246, in CapitalExpenditure_Summary
    for key, value in i.items():
AttributeError: 'str' object has no attribute 'items'
[2024-06-13 10:01:04,909] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:01:04 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:01:04,912] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.41, 127.0.0.1:57614]
[2024-06-13 10:01:55,125] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:57710]
[2024-06-13 10:01:55,204] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:57710]
[2024-06-13 10:01:55,227] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:57710]
[2024-06-13 10:01:55,237] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:57714]
[2024-06-13 10:01:55,270] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:57714]
[2024-06-13 10:01:55,275] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:57710]
[2024-06-13 10:01:55,285] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:57710]
[2024-06-13 10:01:55,403] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:01:55 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:01:55,418] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:57710]
[2024-06-13 10:01:55,464] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:57711]
[2024-06-13 10:01:58,305] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:01:58 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:01:58,322] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:57711]
[2024-06-13 10:03:54,572] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:57914]
[2024-06-13 10:03:54,654] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:57914]
[2024-06-13 10:03:54,672] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:57914]
[2024-06-13 10:03:54,690] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:57917]
[2024-06-13 10:03:54,724] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:57917]
[2024-06-13 10:03:54,741] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:57914]
[2024-06-13 10:03:54,746] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:57915]
[2024-06-13 10:03:54,881] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:03:54 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:03:54,893] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:57915]
[2024-06-13 10:03:54,922] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:57914]
[2024-06-13 10:03:57,954] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 303, in CapitalExpenditure_Summary
    for i in CapitalExpenditure.objects.filter(**Check_dic_CapitalExpenditure):
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'Year' into field. Choices are: Acceptance_Month, Accounting_Subjects, Annual_Increase_PerYear, Applicable_Scope, Application_Department, Attribute_Code, Automated_Or_Not, BudgetCode, Budget_Quantity, Cash_Inflows_PerYear, Current_Situation, Customer, Depreciation_Months, Device_Name, Entry_Amount, Equivalent_To_RMB, Estimated_Original_Currency, Estimated_Original_Price, Investment_Benefits_PerYear, Investment_Nature, Investment_Purpose, Investment_Purpose_Des, Payback_Period, Payment_Terms, PlanYear, Potential_Issues, Potential_Issues_Des, Project_Code, Specifications, Subscription_Amount, Subscription_Quantity, Subscription_Status, Tighten_Expenses, Usage_Description, id
[2024-06-13 10:03:57,956] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:03:57 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:03:57,958] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.18, 127.0.0.1:57914]
[2024-06-13 10:04:42,815] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.56, 127.0.0.1:57914]
[2024-06-13 10:04:42,968] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.07, 127.0.0.1:57914]
[2024-06-13 10:04:42,992] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.02, 127.0.0.1:57914]
[2024-06-13 10:04:43,113] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:57914]
[2024-06-13 10:06:52,536] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.10, 127.0.0.1:58213]
[2024-06-13 10:06:52,615] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:58213]
[2024-06-13 10:06:52,632] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:58213]
[2024-06-13 10:06:52,649] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:58216]
[2024-06-13 10:06:52,682] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:58216]
[2024-06-13 10:06:52,697] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:58213]
[2024-06-13 10:06:52,700] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:58214]
[2024-06-13 10:06:52,828] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:06:52 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:06:52,838] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:58214]
[2024-06-13 10:06:52,883] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:58213]
[2024-06-13 10:06:56,304] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:06:56 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:06:56,314] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:58213]
[2024-06-13 10:12:44,140] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.09, 127.0.0.1:54268]
[2024-06-13 10:12:44,223] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:54268]
[2024-06-13 10:12:44,250] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:54268]
[2024-06-13 10:12:44,295] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:54272]
[2024-06-13 10:12:44,330] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:54272]
[2024-06-13 10:12:44,417] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:54268]
[2024-06-13 10:12:44,424] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:54268]
[2024-06-13 10:12:44,614] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:12:44 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:12:44,642] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:54268]
[2024-06-13 10:12:44,681] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.09, 127.0.0.1:54269]
[2024-06-13 10:12:48,017] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:12:48 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:12:48,030] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:54269]
[2024-06-13 10:12:52,364] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [1.03, 127.0.0.1:54269]
[2024-06-13 10:12:52,790] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.34, 127.0.0.1:54269]
[2024-06-13 10:12:52,845] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:54269]
[2024-06-13 10:12:53,027] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:54269]
[2024-06-13 10:16:37,345] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:16:37 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:16:37,358] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:54259]
[2024-06-13 10:19:09,953] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:19:09 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:19:09,977] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:54694]
[2024-06-13 10:25:37,390] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1735, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1473, in setup_joins
    names[:pivot], opts, allow_many, fail_on_missing=True,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'Year' into field. Choices are: Acceptance_Month, Accounting_Subjects, Annual_Increase_PerYear, Applicable_Scope, Application_Department, Attribute_Code, Automated_Or_Not, BudgetCode, Budget_Quantity, Cash_Inflows_PerYear, Current_Situation, Customer, Depreciation_Months, Device_Name, Entry_Amount, Equivalent_To_RMB, Estimated_Original_Currency, Estimated_Original_Price, Investment_Benefits_PerYear, Investment_Nature, Investment_Purpose, Investment_Purpose_Des, Payback_Period, Payment_Terms, PlanYear, Potential_Issues, Potential_Issues_Des, Project_Code, Specifications, Subscription_Amount, Subscription_Quantity, Subscription_Status, Tighten_Expenses, Usage_Description, id

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 204, in CapitalExpenditure_Summary
    for i in CapitalExpenditure.objects.all().values("Year").distinct().order_by("Year"):
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 750, in values
    clone = self._values(*fields, **expressions)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 745, in _values
    clone.query.set_values(fields)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1987, in set_values
    self.add_fields(field_names, True)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1758, in add_fields
    "Choices are: %s" % (name, ", ".join(names)))
django.core.exceptions.FieldError: Cannot resolve keyword 'Year' into field. Choices are: Acceptance_Month, Accounting_Subjects, Annual_Increase_PerYear, Applicable_Scope, Application_Department, Attribute_Code, Automated_Or_Not, BudgetCode, Budget_Quantity, Cash_Inflows_PerYear, Current_Situation, Customer, Depreciation_Months, Device_Name, Entry_Amount, Equivalent_To_RMB, Estimated_Original_Currency, Estimated_Original_Price, Investment_Benefits_PerYear, Investment_Nature, Investment_Purpose, Investment_Purpose_Des, Payback_Period, Payment_Terms, PlanYear, Potential_Issues, Potential_Issues_Des, Project_Code, Specifications, Subscription_Amount, Subscription_Quantity, Subscription_Status, Tighten_Expenses, Usage_Description, id
[2024-06-13 10:25:37,403] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:25:37 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:25:37,408] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.53, 127.0.0.1:58394]
[2024-06-13 10:26:32,493] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.52, 127.0.0.1:58394]
[2024-06-13 10:26:32,626] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.05, 127.0.0.1:58394]
[2024-06-13 10:26:32,681] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:58394]
[2024-06-13 10:26:32,825] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:58394]
[2024-06-13 10:28:13,285] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:58769]
[2024-06-13 10:28:13,376] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:58769]
[2024-06-13 10:28:13,394] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:58769]
[2024-06-13 10:28:13,447] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:58772]
[2024-06-13 10:28:13,487] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:58772]
[2024-06-13 10:28:13,563] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:58769]
[2024-06-13 10:28:13,567] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:58770]
[2024-06-13 10:28:13,719] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:28:13 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:28:13,728] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:58770]
[2024-06-13 10:28:13,771] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.08, 127.0.0.1:58769]
[2024-06-13 10:28:34,711] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:58833]
[2024-06-13 10:28:34,784] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:58833]
[2024-06-13 10:28:34,824] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:58833]
[2024-06-13 10:28:34,883] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:58837]
[2024-06-13 10:28:34,913] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:58837]
[2024-06-13 10:28:35,006] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:58833]
[2024-06-13 10:28:35,015] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:58833]
[2024-06-13 10:28:35,181] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:28:35 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:28:35,192] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:58833]
[2024-06-13 10:28:35,249] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.09, 127.0.0.1:58830]
[2024-06-13 10:28:40,159] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:28:40 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:28:40,172] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:58830]
[2024-06-13 10:29:19,251] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:29:19 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:29:19,262] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:58830]
[2024-06-13 10:33:10,057] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [1.06, 127.0.0.1:59298]
[2024-06-13 10:33:10,274] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.12, 127.0.0.1:59298]
[2024-06-13 10:33:10,337] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:59298]
[2024-06-13 10:33:10,459] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:59298]
[2024-06-13 10:33:51,965] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:59298]
[2024-06-13 10:33:51,976] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:58837]
[2024-06-13 10:33:52,056] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:59298]
[2024-06-13 10:33:52,086] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:59298]
[2024-06-13 10:33:52,091] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:59367]
[2024-06-13 10:33:52,095] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:59367]
[2024-06-13 10:33:52,142] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:59298]
[2024-06-13 10:33:52,157] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.02, 127.0.0.1:59299]
[2024-06-13 10:33:52,302] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:33:52 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:33:52,314] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:59299]
[2024-06-13 10:33:52,347] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:59298]
[2024-06-13 10:33:56,888] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:33:56 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:33:56,904] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:59298]
[2024-06-13 10:35:50,158] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:59558]
[2024-06-13 10:35:50,166] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:59367]
[2024-06-13 10:35:50,247] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:59558]
[2024-06-13 10:35:50,261] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:59558]
[2024-06-13 10:35:50,283] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:59561]
[2024-06-13 10:35:50,284] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:59561]
[2024-06-13 10:35:50,329] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:59558]
[2024-06-13 10:35:50,337] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:59558]
[2024-06-13 10:35:50,476] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:35:50 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:35:50,487] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:59558]
[2024-06-13 10:35:50,535] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:59559]
[2024-06-13 10:35:54,846] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:35:54 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:35:54,873] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:59559]
[2024-06-13 10:38:38,242] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:38:38 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:38:38,273] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:59840]
[2024-06-13 10:41:29,985] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:60066]
[2024-06-13 10:41:30,070] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:60066]
[2024-06-13 10:41:30,095] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:60066]
[2024-06-13 10:41:30,103] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:60077]
[2024-06-13 10:41:30,135] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:60077]
[2024-06-13 10:41:30,148] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:60066]
[2024-06-13 10:41:30,153] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:60067]
[2024-06-13 10:41:30,285] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:41:30 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:41:30,295] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:60067]
[2024-06-13 10:41:30,326] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:60066]
[2024-06-13 10:41:32,349] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:41:32 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:41:32,359] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:60066]
[2024-06-13 10:43:20,501] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:43:20 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:43:20,532] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:60233]
[2024-06-13 10:58:33,459] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 10:58:33 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 10:58:33,489] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.10, 127.0.0.1:57121]
[2024-06-13 11:05:39,723] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [0.61, 127.0.0.1:52665]
[2024-06-13 11:05:39,745] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.63, 127.0.0.1:52667]
[2024-06-13 11:05:39,761] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.02, 127.0.0.1:52665]
[2024-06-13 11:09:11,798] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/1266/change/ 200 [0.18, 127.0.0.1:53067]
[2024-06-13 11:09:11,817] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/css/forms.css 200 [0.01, 127.0.0.1:53067]
[2024-06-13 11:09:11,822] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/img/icon-addlink.svg 200 [0.01, 127.0.0.1:53069]
[2024-06-13 11:09:11,829] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/img/icon-changelink.svg 200 [0.02, 127.0.0.1:53070]
[2024-06-13 11:09:11,854] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:53071]
[2024-06-13 11:09:11,860] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/css/widgets.css 200 [0.03, 127.0.0.1:53070]
[2024-06-13 11:09:11,866] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/img/icon-deletelink.svg 200 [0.03, 127.0.0.1:53069]
[2024-06-13 11:09:11,873] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/change_form.js 200 [0.04, 127.0.0.1:53067]
[2024-06-13 11:09:11,881] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/admin/js/prepopulate_init.js 200 [0.01, 127.0.0.1:53067]
[2024-06-13 11:09:26,703] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:53067]
[2024-06-13 11:09:28,022] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:53067]
[2024-06-13 11:09:46,780] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [1.17, 127.0.0.1:53067]
[2024-06-13 11:09:46,832] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.03, 127.0.0.1:53067]
[2024-06-13 11:10:24,211] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/1283/change/ 200 [0.19, 127.0.0.1:53067]
[2024-06-13 11:10:24,261] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:53067]
[2024-06-13 11:10:34,173] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [1.19, 127.0.0.1:53067]
[2024-06-13 11:10:34,207] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.02, 127.0.0.1:53067]
[2024-06-13 11:15:59,028] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.54, 127.0.0.1:65346]
[2024-06-13 11:15:59,588] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [0.50, 127.0.0.1:65346]
[2024-06-13 11:15:59,639] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:65346]
[2024-06-13 11:15:59,834] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:65346]
[2024-06-13 11:16:39,301] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/1223/change/ 200 [0.17, 127.0.0.1:65346]
[2024-06-13 11:16:39,348] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:65346]
[2024-06-13 11:16:43,550] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [1.25, 127.0.0.1:65346]
[2024-06-13 12:55:08,587] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.13, 127.0.0.1:54240]
[2024-06-13 12:55:08,696] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:54243]
[2024-06-13 12:55:08,764] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:54243]
[2024-06-13 12:55:08,855] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 12:55:08 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 12:55:08,866] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:54240]
[2024-06-13 12:55:08,932] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:54240]
[2024-06-13 12:55:18,222] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /PersonalInfo/PersonalInfo-search/ 200 [0.06, 127.0.0.1:54240]
[2024-06-13 12:55:18,241] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:54243]
[2024-06-13 12:55:18,275] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:54263]
[2024-06-13 12:55:18,277] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:54263]
[2024-06-13 12:55:19,322] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 12:55:19 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /PersonalInfo/PersonalInfo-search/ POST
[2024-06-13 12:55:19,352] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /PersonalInfo/PersonalInfo-search/ 200 [0.99, 127.0.0.1:54240]
[2024-06-13 12:55:19,390] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [1.02, 127.0.0.1:54241]
[2024-06-13 12:55:25,260] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /PersonalInfo/Infos-upload/ 200 [0.03, 127.0.0.1:54241]
[2024-06-13 12:55:25,284] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/css/b_upload.css 200 [0.01, 127.0.0.1:54241]
[2024-06-13 12:55:25,300] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/js/xlsx/xlsx_pop.js 200 [0.01, 127.0.0.1:54240]
[2024-06-13 12:55:25,303] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:54263]
[2024-06-13 12:55:25,306] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/js/xlsx/xlsx_pop1.js 200 [0.01, 127.0.0.1:54279]
[2024-06-13 12:55:25,312] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/js/xlsx/xlsx_pop2.js 200 [0.02, 127.0.0.1:54241]
[2024-06-13 12:55:25,318] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/js/xlsx/xlsx_pop3.js 200 [0.02, 127.0.0.1:54280]
[2024-06-13 12:55:25,323] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/js/xlsx/xlsx_pop4.js 200 [0.03, 127.0.0.1:54281]
[2024-06-13 12:55:25,329] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/bootstrap-3.3.7-dist/js/bootstrap.min.js 200 [0.03, 127.0.0.1:54282]
[2024-06-13 12:55:25,338] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:54283]
[2024-06-13 12:55:25,341] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:54283]
[2024-06-13 12:55:25,394] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:54282]
[2024-06-13 12:55:29,378] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /static/src/modelfiles/Department.xlsx 200 [0.01, 127.0.0.1:54282]
[2024-06-13 13:07:10,815] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 49, in CapitalExpenditure_Summary
    for i in CapitalExpenditure.objects.annotate(first_seven=Substr(Lower('Application_Department'), 1, 7)).distinct('first_seven'):
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 268, in __iter__
    self._fetch_all()
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 1186, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 54, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\compiler.py", line 1052, in execute_sql
    sql, params = self.as_sql()
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\compiler.py", line 472, in as_sql
    distinct_params,
  File "C:\Python372\mecheck\lib\site-packages\django\db\backends\base\operations.py", line 171, in distinct_sql
    raise NotSupportedError('DISTINCT ON fields is not supported by this database backend')
django.db.utils.NotSupportedError: DISTINCT ON fields is not supported by this database backend
[2024-06-13 13:07:10,822] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.12, 127.0.0.1:61136]
[2024-06-13 13:07:10,930] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:61136]
[2024-06-13 13:08:13,070] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 51, in CapitalExpenditure_Summary
    CustomerOption.append(Departments.objects.filter(Department_Code__contains=i['first_seven'])).first().Customer
AttributeError: 'NoneType' object has no attribute 'first'
[2024-06-13 13:08:13,078] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.12, 127.0.0.1:54178]
[2024-06-13 13:08:13,154] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:54178]
[2024-06-13 13:19:03,987] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:19:03 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /PersonalInfo/Infos-upload/ POST
[2024-06-13 13:19:03,995] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /PersonalInfo/Infos-upload/ 200 [0.51, 127.0.0.1:61384]
[2024-06-13 13:19:15,592] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 51, in CapitalExpenditure_Summary
    CustomerOption.append(Departments.objects.filter(Department_Code__contains=i['first_seven'])).first().Customer
AttributeError: 'NoneType' object has no attribute 'first'
[2024-06-13 13:19:15,594] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.05, 127.0.0.1:61384]
[2024-06-13 13:19:15,649] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:61384]
[2024-06-13 13:19:59,525] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 50, in CapitalExpenditure_Summary
    print(i['first_seven'],CustomerOption.append(Departments.objects.filter(Department_Code__contains=i['first_seven'])))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:19:59,533] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.13, 127.0.0.1:61464]
[2024-06-13 13:19:59,601] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:61464]
[2024-06-13 13:20:45,373] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 50, in CapitalExpenditure_Summary
    print(i['first_seven'],Departments.objects.filter(Department_Code__contains=i['first_seven']))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:20:45,382] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.15, 127.0.0.1:61531]
[2024-06-13 13:20:45,520] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:61531]
[2024-06-13 13:20:55,562] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.51, 127.0.0.1:61531]
[2024-06-13 13:20:56,243] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [0.59, 127.0.0.1:61531]
[2024-06-13 13:20:56,270] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.02, 127.0.0.1:61531]
[2024-06-13 13:20:56,532] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:61531]
[2024-06-13 13:26:01,907] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 50, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:26:01,916] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.15, 127.0.0.1:60477]
[2024-06-13 13:26:02,009] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:60477]
[2024-06-13 13:27:32,603] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 50, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']).first())
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:27:32,612] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.12, 127.0.0.1:57909]
[2024-06-13 13:27:32,715] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:57909]
[2024-06-13 13:27:58,295] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 50, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']).first().Customer)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:27:58,302] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.14, 127.0.0.1:57948]
[2024-06-13 13:27:58,364] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:57948]
[2024-06-13 13:28:22,254] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 50, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']).first().Customer)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:28:22,262] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.13, 127.0.0.1:58931]
[2024-06-13 13:28:22,324] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.02, 127.0.0.1:58931]
[2024-06-13 13:28:54,157] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 51, in CapitalExpenditure_Summary
    CustomerOption.append(Departments.objects.filter(Department_Code__contains=i['first_seven']).first().Customer)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:28:54,166] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.13, 127.0.0.1:58996]
[2024-06-13 13:28:54,222] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.02, 127.0.0.1:58996]
[2024-06-13 13:32:32,471] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 53, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']).first().Customer)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:32:32,479] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.14, 127.0.0.1:60397]
[2024-06-13 13:32:32,548] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:60397]
[2024-06-13 13:39:55,386] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 53, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']).Customer)
AttributeError: 'QuerySet' object has no attribute 'Customer'
[2024-06-13 13:39:55,392] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.12, 127.0.0.1:63614]
[2024-06-13 13:39:55,460] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:63614]
[2024-06-13 13:41:12,552] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 53, in CapitalExpenditure_Summary
    print(i['first_seven'], Departments.objects.filter(Department_Code__contains=i['first_seven']).first())
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1225, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1101, in build_lookup
    raise ValueError("Cannot use None as a query value")
ValueError: Cannot use None as a query value
[2024-06-13 13:41:12,560] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.13, 127.0.0.1:63742]
[2024-06-13 13:41:12,614] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.02, 127.0.0.1:63742]
[2024-06-13 13:42:54,761] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:63887]
[2024-06-13 13:42:54,903] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:63887]
[2024-06-13 13:42:54,949] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:63887]
[2024-06-13 13:42:54,955] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:63892]
[2024-06-13 13:42:54,993] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:63892]
[2024-06-13 13:42:55,010] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:63887]
[2024-06-13 13:42:55,015] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:63882]
[2024-06-13 13:42:55,195] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:42:55 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 13:42:55,207] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:63882]
[2024-06-13 13:42:55,244] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:63887]
[2024-06-13 13:43:08,059] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:63887]
[2024-06-13 13:43:08,065] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:63892]
[2024-06-13 13:43:08,147] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:63887]
[2024-06-13 13:43:08,161] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:63887]
[2024-06-13 13:43:08,167] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:63920]
[2024-06-13 13:43:08,170] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:63920]
[2024-06-13 13:43:08,211] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:63887]
[2024-06-13 13:43:08,215] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:63882]
[2024-06-13 13:43:08,358] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:43:08 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 13:43:08,367] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:63882]
[2024-06-13 13:43:08,407] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:63887]
[2024-06-13 13:43:30,330] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.87, 127.0.0.1:63887]
[2024-06-13 13:43:32,450] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/PersonalInfo/departments/ 200 [2.02, 127.0.0.1:63887]
[2024-06-13 13:43:32,493] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.03, 127.0.0.1:63887]
[2024-06-13 13:43:32,778] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.04, 127.0.0.1:63887]
[2024-06-13 13:44:53,667] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:64077]
[2024-06-13 13:44:53,751] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:64077]
[2024-06-13 13:44:53,765] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:64077]
[2024-06-13 13:44:53,781] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64080]
[2024-06-13 13:44:53,811] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64080]
[2024-06-13 13:44:53,825] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:64077]
[2024-06-13 13:44:53,831] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:64078]
[2024-06-13 13:44:53,977] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:44:53 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 13:44:53,986] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:64078]
[2024-06-13 13:44:54,027] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:64077]
[2024-06-13 13:45:14,471] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:64125]
[2024-06-13 13:45:15,269] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:64114]
[2024-06-13 13:45:15,364] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:64114]
[2024-06-13 13:45:15,437] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:64114]
[2024-06-13 13:45:15,453] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64129]
[2024-06-13 13:45:15,490] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64129]
[2024-06-13 13:45:15,502] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:64114]
[2024-06-13 13:45:15,516] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:64114]
[2024-06-13 13:45:15,690] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:45:15 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 13:45:15,723] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:64114]
[2024-06-13 13:45:15,749] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:64126]
[2024-06-13 13:51:30,801] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:64616]
[2024-06-13 13:51:30,884] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:64616]
[2024-06-13 13:51:30,917] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:64616]
[2024-06-13 13:51:30,960] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64619]
[2024-06-13 13:51:30,993] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64619]
[2024-06-13 13:51:31,060] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:64616]
[2024-06-13 13:51:31,068] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:64616]
[2024-06-13 13:51:31,200] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:51:31 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 13:51:31,212] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:64616]
[2024-06-13 13:51:31,258] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:64617]
[2024-06-13 13:52:00,127] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:64672]
[2024-06-13 13:52:00,217] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:64672]
[2024-06-13 13:52:00,239] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:64672]
[2024-06-13 13:52:00,288] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64677]
[2024-06-13 13:52:00,315] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64677]
[2024-06-13 13:52:00,392] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:64672]
[2024-06-13 13:52:00,401] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:64672]
[2024-06-13 13:52:00,545] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 13:52:00 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 13:52:00,556] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:64672]
[2024-06-13 13:52:00,602] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:64673]
[2024-06-13 14:40:06,344] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/CapitalExpenditure/capitalexpenditure/ 200 [0.84, 127.0.0.1:50009]
[2024-06-13 14:40:06,397] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.86, 127.0.0.1:50010]
[2024-06-13 14:40:06,431] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.07, 127.0.0.1:50009]
[2024-06-13 15:28:19,427] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:62032]
[2024-06-13 15:28:19,508] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:62032]
[2024-06-13 15:28:19,541] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:62032]
[2024-06-13 15:28:19,584] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:62035]
[2024-06-13 15:28:19,620] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:62035]
[2024-06-13 15:28:19,687] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:62032]
[2024-06-13 15:28:19,694] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.01, 127.0.0.1:62033]
[2024-06-13 15:28:19,834] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:28:19 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:28:19,846] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:62033]
[2024-06-13 15:28:19,895] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:62032]
[2024-06-13 15:28:42,411] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:62095]
[2024-06-13 15:28:42,500] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:62095]
[2024-06-13 15:28:42,527] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:62095]
[2024-06-13 15:28:42,576] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:62099]
[2024-06-13 15:28:42,602] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:62099]
[2024-06-13 15:28:42,681] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:62095]
[2024-06-13 15:28:42,686] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:62094]
[2024-06-13 15:28:42,826] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:28:42 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:28:42,836] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:62094]
[2024-06-13 15:28:42,877] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.08, 127.0.0.1:62095]
[2024-06-13 15:28:51,266] [log.py:228] [log:log_response] [ERROR]- Internal Server Error: /CapitalExpenditure/CapitalExpenditure_Summary/
Traceback (most recent call last):
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python372\mecheck\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\djangoproject\Reliability_Row_data\CapitalExpenditure\views.py", line 387, in CapitalExpenditure_Summary
    Application_Department__contains=DepartmentSeven,).aggregate(total=Sum('Budget_Quantity'))[
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "C:\Python372\mecheck\lib\site-packages\django\db\models\sql\query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'Year' into field. Choices are: Acceptance_Month, Accounting_Subjects, Annual_Increase_PerYear, Applicable_Scope, Application_Department, Attribute_Code, Automated_Or_Not, BudgetCode, Budget_Quantity, Cash_Inflows_PerYear, Current_Situation, Customer, Depreciation_Months, Device_Name, Entry_Amount, Equivalent_To_RMB, Estimated_Original_Currency, Estimated_Original_Price, Investment_Benefits_PerYear, Investment_Nature, Investment_Purpose, Investment_Purpose_Des, Payback_Period, Payment_Terms, PlanYear, Potential_Issues, Potential_Issues_Des, Project_Code, Specifications, Subscription_Amount, Subscription_Quantity, Subscription_Status, Tighten_Expenses, Usage_Description, id
[2024-06-13 15:28:51,274] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:28:51 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:28:51,276] [runserver.py:154] [runserver:log_action] [ERROR]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 500 [0.43, 127.0.0.1:62095]
[2024-06-13 15:31:32,502] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:62400]
[2024-06-13 15:31:32,604] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:62400]
[2024-06-13 15:31:32,638] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:62400]
[2024-06-13 15:31:32,691] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:62403]
[2024-06-13 15:31:32,724] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:62403]
[2024-06-13 15:31:32,788] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:62400]
[2024-06-13 15:31:32,795] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:62397]
[2024-06-13 15:31:32,941] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:31:32 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:31:32,951] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.03, 127.0.0.1:62397]
[2024-06-13 15:31:32,994] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.05, 127.0.0.1:62400]
[2024-06-13 15:31:37,757] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:31:37 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:31:37,765] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:62400]
[2024-06-13 15:33:00,343] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:33:00 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:33:00,358] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:62528]
[2024-06-13 15:33:41,058] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:33:41 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:33:41,094] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.10, 127.0.0.1:62596]
[2024-06-13 15:39:11,522] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:39:11 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:39:11,552] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.08, 127.0.0.1:60028]
[2024-06-13 15:40:20,574] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:60120]
[2024-06-13 15:40:20,653] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.00, 127.0.0.1:60120]
[2024-06-13 15:40:20,687] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:60120]
[2024-06-13 15:40:20,733] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:60125]
[2024-06-13 15:40:20,761] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:60125]
[2024-06-13 15:40:20,804] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.00, 127.0.0.1:60120]
[2024-06-13 15:40:20,837] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:60120]
[2024-06-13 15:40:20,986] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:40:20 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:40:20,996] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:60120]
[2024-06-13 15:40:21,055] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:60121]
[2024-06-13 15:40:24,742] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:40:24 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:40:24,795] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.09, 127.0.0.1:60121]
[2024-06-13 15:46:30,189] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:46:30 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:46:30,209] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.07, 127.0.0.1:60646]
[2024-06-13 15:46:32,217] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.04, 127.0.0.1:60646]
[2024-06-13 15:46:32,301] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:60646]
[2024-06-13 15:46:32,333] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.00, 127.0.0.1:60646]
[2024-06-13 15:46:32,378] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:60652]
[2024-06-13 15:46:32,408] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:60652]
[2024-06-13 15:46:32,482] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.01, 127.0.0.1:60646]
[2024-06-13 15:46:32,490] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.00, 127.0.0.1:60646]
[2024-06-13 15:46:32,631] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:46:32 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:46:32,642] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.02, 127.0.0.1:60646]
[2024-06-13 15:46:32,694] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:60650]
[2024-06-13 15:46:34,282] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-13 15:46:34 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-13 15:46:34,291] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:60650]
[2024-06-14 07:54:43,614] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:60652]
[2024-06-14 07:54:53,462] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET / 302 [0.09, 127.0.0.1:64717]
[2024-06-14 07:54:53,470] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /login/ 200 [0.00, 127.0.0.1:64717]
[2024-06-14 07:54:53,601] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 302 [0.01, 127.0.0.1:64717]
[2024-06-14 07:54:53,607] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /login/ 200 [0.00, 127.0.0.1:64717]
[2024-06-14 07:54:57,414] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-14 07:54:57 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /login/ POST
[2024-06-14 07:54:57,431] [runserver.py:147] [runserver:log_action] [INFO]- HTTP POST /login/?next= 302 [0.06, 127.0.0.1:64739]
[2024-06-14 07:54:57,455] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /login/ 302 [0.02, 127.0.0.1:64739]
[2024-06-14 07:54:57,668] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /index/ 200 [0.21, 127.0.0.1:64739]
[2024-06-14 07:54:57,832] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64744]
[2024-06-14 07:54:57,839] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64744]
[2024-06-14 07:54:58,014] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:64739]
[2024-06-14 07:55:05,427] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.06, 127.0.0.1:64739]
[2024-06-14 07:55:05,484] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64759]
[2024-06-14 07:55:05,486] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64759]
[2024-06-14 07:55:05,499] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64744]
[2024-06-14 07:55:05,660] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-14 07:55:05 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-14 07:55:05,668] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:64739]
[2024-06-14 07:55:05,706] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.07, 127.0.0.1:64740]
[2024-06-14 07:55:26,267] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /admin/ 302 [0.13, 127.0.0.1:64740]
[2024-06-14 07:55:26,309] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/login/?next=/admin/ 200 [0.04, 127.0.0.1:64740]
[2024-06-14 07:55:26,341] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64759]
[2024-06-14 07:55:26,440] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:64740]
[2024-06-14 07:55:29,500] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-14 07:55:29 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /admin/login/ POST
[2024-06-14 07:55:29,510] [runserver.py:147] [runserver:log_action] [INFO]- HTTP POST /admin/login/?next=/admin/ 302 [0.24, 127.0.0.1:64740]
[2024-06-14 07:55:30,230] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/ 200 [0.72, 127.0.0.1:64740]
[2024-06-14 07:55:30,379] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:64740]
[2024-06-14 07:55:36,048] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/app01/menu/ 200 [0.25, 127.0.0.1:64740]
[2024-06-14 07:55:36,084] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.28, 127.0.0.1:64739]
[2024-06-14 07:55:36,111] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:64740]
[2024-06-14 07:55:43,389] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/app01/menu/41/change/ 200 [0.14, 127.0.0.1:64740]
[2024-06-14 07:55:43,430] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.03, 127.0.0.1:64740]
[2024-06-14 07:55:58,889] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-14 07:55:58 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /admin/app01/menu/41/change/ POST
[2024-06-14 07:55:58,898] [runserver.py:147] [runserver:log_action] [INFO]- HTTP POST /admin/app01/menu/41/change/ 302 [0.09, 127.0.0.1:64740]
[2024-06-14 07:55:59,100] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/app01/menu/ 200 [0.20, 127.0.0.1:64740]
[2024-06-14 07:55:59,134] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /admin/jsi18n/ 200 [0.02, 127.0.0.1:64740]
[2024-06-14 07:56:02,590] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET / 200 [0.04, 127.0.0.1:64740]
[2024-06-14 07:56:02,660] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64859]
[2024-06-14 07:56:02,663] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64859]
[2024-06-14 07:56:02,722] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.03, 127.0.0.1:64740]
[2024-06-14 07:56:08,311] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /logout/ 302 [0.02, 127.0.0.1:64740]
[2024-06-14 07:56:08,319] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /login/ 200 [0.01, 127.0.0.1:64740]
[2024-06-14 07:56:08,325] [runserver.py:160] [runserver:log_action] [INFO]- WebSocket DISCONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64859]
[2024-06-14 07:56:08,374] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 302 [0.01, 127.0.0.1:64740]
[2024-06-14 07:56:08,385] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /login/ 200 [0.01, 127.0.0.1:64740]
[2024-06-14 07:56:10,875] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-14 07:56:10 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /login/ POST
[2024-06-14 07:56:10,885] [runserver.py:147] [runserver:log_action] [INFO]- HTTP POST /login/?next= 302 [0.06, 127.0.0.1:64740]
[2024-06-14 07:56:10,905] [runserver.py:147] [runserver:log_action] [INFO]- HTTP GET /login/ 302 [0.02, 127.0.0.1:64740]
[2024-06-14 07:56:10,962] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:64740]
[2024-06-14 07:56:11,029] [runserver.py:162] [runserver:log_action] [INFO]- WebSocket HANDSHAKING /ws/msg/Reliability_Row_data/ [127.0.0.1:64876]
[2024-06-14 07:56:11,035] [runserver.py:158] [runserver:log_action] [INFO]- WebSocket CONNECT /ws/msg/Reliability_Row_data/ [127.0.0.1:64876]
[2024-06-14 07:56:11,183] [UserIP.py:90] [UserIP:process_response] [INFO]- 2024-06-14 07:56:11 C1010S3>Edwin None>None>None 127.0.0.1 127.0.0.1 /CapitalExpenditure/CapitalExpenditure_Summary/ POST
[2024-06-14 07:56:11,195] [runserver.py:141] [runserver:log_action] [INFO]- HTTP POST /CapitalExpenditure/CapitalExpenditure_Summary/ 200 [0.05, 127.0.0.1:64740]
[2024-06-14 07:56:11,235] [runserver.py:141] [runserver:log_action] [INFO]- HTTP GET /favicon.ico 200 [0.06, 127.0.0.1:64739]
[2024-06-14 07:59:03,608] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/lib/bootstrap.min.js.map 404 [0.01, 127.0.0.1:65103]
[2024-06-14 07:59:03,620] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/axios.min.map 404 [0.02, 127.0.0.1:65104]
[2024-06-14 07:59:03,628] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/js/xlsx/FileSaver.min.js.map 404 [0.03, 127.0.0.1:65105]
[2024-06-14 07:59:03,643] [runserver.py:149] [runserver:log_action] [WARNING]- HTTP GET /static/css/bootstrap.min.css.map 404 [0.01, 127.0.0.1:65105]
